packages/moby-containerd/mapping.go

```diff
@@ -20,141 +22,23 @@ var (
 	rpmPreRm string
 	//go:embed postinstall/rpm/upgrade
 	rpmUpgrade string
+)
 
-	Archives = map[string]archive.Archive{
-		"bookworm": DebArchive,
-		"buster":   DebArchive,
-		"bullseye": DebArchive,
-		"bionic":   DebArchive,
-		"focal":    DebArchive,
-		"centos7":  RPMArchive,
-		"rhel8":    RPMArchive,
-		"rhel9":    RPMArchive,
-		"windows":  BaseArchive,
-		"jammy":    DebArchive,
-		"mariner2": MarinerArchive,
-	}
-
-	BaseArchive = archive.Archive{
-		Name:    "moby-containerd",
-		Webpage: "https://github.com/containerd/containerd",
-		Files: []archive.File{
-			{Source: "/build/src/bin", Dest: "usr/bin"},
-			{Source: "/build/man", Dest: "/usr/share/man"},
-			{Source: "/build/legal/LICENSE", Dest: "/usr/share/doc/moby-containerd/LICENSE"},
-			{Source: "/build/legal/NOTICE", Dest: "/usr/share/doc/moby-containerd/NOTICE.gz", Compress: true},
-		},
-		Systemd: []archive.Systemd{
-			{Source: "/build/src/containerd.service", Dest: "lib/systemd/system/containerd.service"},
-		},
-		Binaries: []string{
-			"/build/src/bin/containerd",
-			"/build/src/bin/containerd-shim",
-			"/build/src/bin/containerd-shim-runc-v1",
-			"/build/src/bin/containerd-shim-runc-v2",
-			"/build/src/bin/containerd-stress",
-		},
-		WinBinaries: []string{
-			"/build/src/bin/containerd.exe",
-			"/build/src/bin/containerd-shim-runhcs-v1.exe",
-			"/build/src/bin/ctr.exe",
-		},
-		Description: `Industry-standard container runtime
- containerd is an industry-standard container runtime with an emphasis on
- simplicity, robustness and portability. It is available as a daemon for Linux
- and Windows, which can manage the complete container lifecycle of its host
- system: image transfer and storage, container execution and supervision,
- low-level storage and network attachments, etc.
- .
- containerd is designed to be embedded into a larger system, rather than being
- used directly by developers or end-users.`,
-	}
+var verPrefixRegex = regexp.MustCompile(`^(\d+)\.(\d+)\.?\d+?`)
```

Comment:
We are already importing the `semver` package, can we use that instead?

Reply:
Its not really a semver string, but in this case we don't care about anything other than major.minor, so for now we can trim off the non semver'd bits. I'll update.

Reply:
Done.